make/variables.sh

```diff
@@ -80,6 +80,9 @@ validate_input "${TFVARS_JSON}"
 # Parse tfvars.json and export variables
 # DNS Provider variable
 export DNS_PROVIDER=$(jq -r '.tetrate.dns_provider // .tetrate.fqdn | select(type == "string") | split(".") | if length > 1 then .[1] else .[0] end | select(. != null) | sub("sandbox"; "gcp")' "${TFVARS_JSON}")
+if [[ "${DNS_PROVIDER}" == aws* ]]; then
```

Comment:
If we update line 82 to looks for specific prefix instead like:
`jq -r '.tetrate.dns_provider // .tetrate.fqdn | select(type == "string") | split(".") | if length > 1 then .[1] else .[0] end | select(. != null) | sub("sandbox"; "gcp") | capture("^(?<cloud>aws|gcp|azure)") | .cloud'`

Would this achieve what your aiming for?  this way we can account for any cloud being a prefix ie: gcp-ce 
WDYT @AlexeySkvortsov

Reply:
@shamusx yes, I like this idea and yes, this achieve what I am aiming for and covers all other cloud prefixes. done.